### PR TITLE
vscode-quarkus 1.13.0 release blog

### PR DESCRIPTION
**Pull request by JessicaJHee**
Add blog post for VSCode Quarkus 1.13.0 Qute features

Signed-off-by: Jessica He <jhe@redhat.com>

### CODE REVIEW

**Comment by JessicaJHee** [comment]
@gsmet Do you mind taking a look at this? Thank you!

File: _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: 'New Features for Qute Templating Engine Support in Quarkus Tools for Visual Studio Code 1.13.0'
+synopsis: Quarkus Tools for Visual Studio Code version 1.13.0 brings many new features in its Qute Templating Engine support.
+date: 2023-04-04
```
Comment (gsmet) [inline _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc:5]: Let's target for a publication tomorrow: you need to update this date and the date of the file accordingly.

File: _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+tags: ide
+author: jhe
+---
+:imagesdir: /assets/images/posts/quarkus-vs-code-1.13.0
+:star: *
+
+Quarkus Tools for Visual Studio Code 1.13.0 has been released on the
+https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[VS Code Marketplace] & https://open-vsx.org/extension/redhat/vscode-quarkus[Open VSX].
```
Comment (gsmet) [inline _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc:13]:
```suggestion
https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus[VS Code Marketplace] and https://open-vsx.org/extension/redhat/vscode-quarkus[Open VSX].
```

File: _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+image::fragmentQuarkusIntegration.gif[alt="Fragment Quarkus Integration"]
+
+== User Tag Support Improvement
+
+User-defined Tags are used to define a tag template. To better support this feature in this release, we added smarter snippets completion, definition, validation, and a Quick Fix to insert required parameters.
+
+=== Smarter Snippet Completion
+
+The snippet completion will now take into consideration the expected content and parameters defined in the user tag. For instance, the input user tag expects the parameter name, which is generated in the snippet with placeholder values. The section tag generated is self-closed if it does not expect any content, while a new line and end tag is generated if the user tag defines some nested content. See this in action in the demo below.
```
Comment (gsmet) [inline _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc:62]: > See this in action in the demo below.

Is the demo missing?

File: _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+== Surround with Command
+
+We have added 3 commands to Qute templates that surround the selected section with comments, unparsed character data, or section tags. The selection range will be adjusted to select the entire section if the command is triggered when the cursor is on the section start or end tag.
+
+image::surroundWithCommand.gif[alt="Surround With Command"]
+
+== Renarde Support
+
+Renarde is a server-side Web Framework based on Quarkus, Qute, Hibernate, and RESTEasy Reactive. In this release, we have added support for this framework both in the Renarde template and Java file.
```
Comment (gsmet) [inline _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc:98]:
```suggestion
Renarde is a server-side Web Framework based on Quarkus, Qute, Hibernate ORM, and RESTEasy Reactive. In this release, we have added support for this framework both in the Renarde template and Java file.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Amazing changelog. I added a few suggestions/questions. Let's try to get it merged by tonight so it gets published tomorrow.

File: _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc [hunk 5]
```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: 'New Features for Qute Templating Engine Support in Quarkus Tools for Visual Studio Code 1.13.0'
+synopsis: Quarkus Tools for Visual Studio Code version 1.13.0 brings many new features in its Qute Templating Engine support.
+date: 2023-04-19
```
Comment (gsmet) [inline _posts/2023-04-12-vscode-quarkus-1.13.0-released.adoc:5]: As mentioned, you also need to rename the file. Apart from that, it looks good.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
@JessicaJHee I'll be happy to tweet something about it if you can write a tweet (or retweet something if you already have posted one)

**Comment by JessicaJHee** [comment]
@gsmet I've made a tweet here: https://twitter.com/JessicaJjhe/status/1648704525409169409?s=20

Thanks very much!

**Pull request merged**